app/assets/javascripts/views/mobile_menu.js

```diff
@@ -0,0 +1,43 @@
+(function(App) {
+
+  'use strict';
+
+  var MobileMenu = Backbone.View.extend({
+
+    el: 'body',
+
+    events: {
+      'click .btn-mobile-menu' : 'onClickToggleMenu',
+      'click .btn-close' : 'onClickToggleMenu'
+    },
+
+    initialize: function() {
+      this.status =  new (Backbone.Model.extend({
+        defaults: {
+          hidden: true
+        }
+      }));
+
+      this._listeners();
+    },
+
+    _listeners: function() {
+      this.status.on('change:hidden', this.changedStatus, this);
```

Comment:
I would call this function with the name of the action what is gonna happen instead of the actions that already happened. So for example, toggleMenuPanel or something like that.